client/js/components/CreateGameDialog.vue

```diff
@@ -63,6 +65,14 @@ export default {
       isRanked: false,
     };
   },
+  watch: {
+    isRanked(isRanked) {
+      this.setRankedPreference(isRanked);
+    },
+  },
+  mounted() {
+    this.isRanked = this.getRankedPreference();
+  },
```

Comment:
Any reason to prefer `mounted()` over `created()`?
```suggestion
  created() {
    this.isRanked = this.getRankedPreference();
  },
```

Reply:
I don't think we should do this. `mounted()` is just as good and while we're not a SSR app, created has the window only in the browser (which would require an `if (window)` check) or something with `process.env.VUE_ENV === 'client'` if we ever looked in to it. We get the same benefit with mounted and the knowledge that window is always available within it.